proto/stride/stakeibc/tx.proto

```diff
@@ -174,57 +200,81 @@ message ValidatorWeight {
   uint64 weight = 2;
 }
 message MsgChangeValidatorWeights {
-  string creator = 1;
+  option (cosmos.msg.v1.signer) = "creator";
+  option (amino.name) = "stakeibc/MsgChangeValidatorWeights";
+
+  string creator = 1 [ (cosmos_proto.scalar) = "cosmos.AddressString" ];
   string host_zone = 2;
   repeated ValidatorWeight validator_weights = 3;
 }
 message MsgChangeValidatorWeightsResponse {}
 
 message MsgDeleteValidator {
-  string creator = 1;
+  option (cosmos.msg.v1.signer) = "creator";
+  option (amino.name) = "stakeibc/MsgDeleteValidator";
+
+  string creator = 1 [ (cosmos_proto.scalar) = "cosmos.AddressString" ];
   string host_zone = 2;
-  string val_addr = 3 [ (cosmos_proto.scalar) = "cosmos.AddressString" ];
+  string val_addr = 3
+      [ (cosmos_proto.scalar) = "cosmos.ValidatorAddressString" ];
 }
 message MsgDeleteValidatorResponse {}
 
 message MsgRestoreInterchainAccount {
-  string creator = 1;
+  option (cosmos.msg.v1.signer) = "creator";
+  option (amino.name) = "stakeibc/MsgRestoreInterchainAcco";
+
+  string creator = 1 [ (cosmos_proto.scalar) = "cosmos.AddressString" ];
   string chain_id = 2;
   string connection_id = 3;
   string account_owner = 4;
 }
 message MsgRestoreInterchainAccountResponse {}
 
 message MsgCloseDelegationChannel {
-  string creator = 1;
+  option (cosmos.msg.v1.signer) = "creator";
+  option (amino.name) = "stakeibc/MsgCloseDelegationChanne";
+
+  string creator = 1 [ (cosmos_proto.scalar) = "cosmos.AddressString" ];
   string chain_id = 2;
 }
 message MsgCloseDelegationChannelResponse {}
 
 message MsgUpdateValidatorSharesExchRate {
-  string creator = 1;
+  option (cosmos.msg.v1.signer) = "creator";
+  option (amino.name) = "stakeibc/MsgUpdateValidatorSharesExchRate";
```

Comment:
I think this message name is too long - can't exceed 39 chars

Reply:
Yeah it's 41 :cry:

Reply:
201c9f1b42ed35c079419bc6d08c4f6129bc22c6?

Reply:
would it make more sense to just abbreviate the amino name, but not the actual message name?

Changing the actual message name seems like it'd be more invasive (would create an indexing inconsistency). That said, I could go either way